OasisDEX/oasis-borrow
# Rename Banner.tsx to Notice.tsx and add new Banner comp (from Design System)

# [Rename Legacy Banner to Notice and add Banner comp from Design System ](https://app.shortcut.com/oazo-apps/story/4918/refactor-banner-component-to-marry-up-with-design-system)
  
Notices are the full width banners that sit ABOVE a vault.

## Changes 👷‍♀️
- Rename Banner to Notice and update naming appropriately through codebase
- Introduce generic Banner component that marries up to Design system https://www.figma.com/file/S6pnwbeRY6T8hfNQKxzp7K/Components?node-id=548%3A3347
- Replace all Banner implementations with the Generic Banner component
  
## How to test 🧪
- Check bonus claim notice
- Check liquidation notices
- Check your viewing someone elses vault notice
- Confirm in-line banners eg auto buy or set up stop loss
